tensorflow/compiler/mlir/quantization/tensorflow/utils/BUILD

```diff
@@ -16,6 +16,8 @@ cc_library(
     name = "fake_quant_utils",
     srcs = ["fake_quant_utils.cc"],
     hdrs = ["fake_quant_utils.h"],
+    # TODO(b/220688154): Restrict the visibility
+    visibility = ["//visibility:public"],
```

Comment:
Just a test. Are you notified by my comment here?

Reply:
@abattery  I suppose you was not notified by my previous comment until this one wgere I've explicitly mentioned you. Right?